jobs-builder/jjb.conf.template

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Jenkins Job Builder configuration file (template).
+#
+[job_builder]
+ignore_cache=True
+keep_descriptions=False
+include_path=.:scripts:~/git/
+recursive=False
+exclude=.*:manual:./development
+allow_duplicates=False
+update=all
+
+[jenkins]
+user=XXX
```

Comment:
is this going to be populated? or the user needs to modify it manually

Reply:
@GabyCT the user needs to modify it. I created it just as an example so that users don't need to read the jenkins job builder documentation in order to use the publish_jobs.sh script.

I plan to have another job on Jenkins which would call `publish_jobs.sh` whenever a file change for `jobs-builder/*` is merged.